blockchain/chainio.go

```diff
@@ -1434,7 +1434,9 @@ func (b *BlockChain) initChainState() error {
 		header := &block.Header
 		node := newBlockNode(header, &state.hash, int64(state.height),
 			ticketsSpentInBlock(dcrutil.NewBlock(&block)),
-			ticketsRevokedInBlock(dcrutil.NewBlock(&block)))
+			ticketsRevokedInBlock(dcrutil.NewBlock(&block)),
+			voteVersionsInBlock(dcrutil.NewBlock(&block),
```

Comment:
I this was already doing it, but can we move the creation of the dcrutil.Block before the function call so it's only done once?  It's fairly expensive to create one.

```Go
	utilBlock := dcrutil.NewBlock(&block)
	header := &block.Header
	node := newBlockNode(header, &state.hash, int64(state.height),
		ticketsSpentInBlock(utilBlock),	ticketsRevokedInBlock(utilBlock),
		voteVersionsInBlock(utilBlock, b.chainParams))
```